examples/search_examples/kibana.json

```diff
@@ -0,0 +1,8 @@
+{
+  "id": "searchExamples",
+  "version": "8.0.0",
+  "server": true,
+  "ui": true,
+  "requiredPlugins": ["navigation", "data"],
+  "optionalPlugins": []
```

Comment:
nit: please add our new  `requiredBundles` field. I see at least `data` should be there